configure.py

```diff
@@ -1,11 +1,24 @@
-# vim: set sts=2 ts=8 sw=2 tw=99 et:
+# vim: set ts=2 sw=2 tw=99 noet:
 import sys
-from ambuild2 import run
+try:
+	from ambuild2 import run, util
+except:
+	try:
+		import ambuild
+		sys.stderr.write('It looks like you have AMBuild 1 installed, but this project uses AMBuild 2.\n')
+		sys.stderr.write('Upgrade to the latest version of AMBuild to continue.\n')
+	except:
+		sys.stderr.write('AMBuild must be installed to build this project.\n')
+		sys.stderr.write('http://www.alliedmods.net/ambuild\n')
+	sys.exit(1)
 
-# Simple extensions do not need to modify this file.
-
-parser = run.BuildParser(sourcePath = sys.path[0], api='2.2')
+# Hack to show a decent upgrade message, which wasn't done until 2.2.
+ambuild_version = getattr(run, 'CURRENT_API', '2.1')
+if ambuild_version.startswith('2.1'):
+	sys.stderr.write("AMBuild 2.2 or higher is required; please update\n")
+	sys.exit(1)
```

Comment:
I don't think any of this is necessary, nobody runs old ambuild anymore. And we already enforce 2.2 through 
`parser = run.BuildParser(sourcePath=sys.path[0], api='2.2')`